app/_src/gateway/production/monitoring/health-check.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Health Check
+content_type: tutorial
+---
+
+This tutorial will guide you through the process of using the new Node Readiness Endpoint, which provides a more reliable and simple way to determine if Kong is ready to serve user requests.
+
+To put it simply, health check endpoint will return a `200 OK` response when Kong is ready, or a `503 Service Unavailable` response when it's not. This is useful for load balancers and other tools that need to monitor the readiness of Kong instances.
+
+Note that the readiness endpoint does not return detailed node health information such as the
+amount of memory Kong node is using. You might want to check out the [`/status` API](/gateway/{{page.kong_version}}/admin-api/#retrieve-node-status) as well.
+
+## Prerequisites
+
+* Kong installed
+* Basic understanding of Kong configuration and deployment modes (Traditional, DB-less, and Hybrid)
+
+## Understanding the Node Readiness Endpoint
+
+Before diving into the steps, it's important to understand the purpose of the Node Readiness Endpoint and how it determines whether a Kong instance is ready or not.
+
+### Traditional mode
+
+In Traditional mode, the conditions that must be fulfilled simultaneously for the traditional mode are: 
+
+1. Successful connection to the database
+2. Successful initial build of all routers for all workers
+3. Successful initial build of all plugin iterators for all workers
+
+### DB-less mode (`data_plane` role)
+
+In DB-less mode, the endpoint returns 200 OK when all of the following conditions are met:
+
+1. Kong has loaded a valid and non-empty config
+2. Successful initial build of all routers for all workers
+3. Successful initial build of all plugin iterators for all workers
+
+## Configuring the Status Endpoint
+
+To configure the Status Endpoint in your Kong instance, follow these steps:
+
+1. Ensure that Kong is setup.
+2. Open the Kong configuration file (e.g., `kong.conf` or `kong.yml`).
+3. Make sure the Status API server is enabled by including the following line:
+
+    ```conf
+    status_listen = 0.0.0.0:8100
+    ```
+
+    This will make the Status API server listen on port `8100` on all interfaces of the system. You can change the port number to any available port on your system.
+
+4. Save the configuration file and restart Kong to apply the changes.
+
+    ```shell
+    kong reload
+    ```
+    
+You can also use `KONG_STATUS_LISTEN` environment variable to configure it.
```

Comment:
This is all subjective, but I feel that these procedural instructions are overly verbose. I understand that it's a tutorial and not a reference, but our audience is someone who is already using Kong, so setting a particular configuration parameter should be well-understood to them.

Here is what I suggest instead:


```suggestion
## Enabling the Status Endpoint

In order to use the Node Readiness Endpoint, you must first enable the Status API Server via the [status_listen](https://docs.konghq.com/gateway/latest/reference/configuration/#status_listen) configuration parameter.

Example:

    ```conf
    # kong.conf
    status_listen = 0.0.0.0:8100
    ```
```